.github/dependabot.yml

```diff
@@ -0,0 +1,20 @@
+version: 2
+updates:
+  # Dependencies listed in go.mod
+  - package-ecosystem: "gomod"
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "weekly"
+    labels:
```

Comment:
we might want to add the ok-to-test label here like CAPI does so we don't have to approve each PR manually for testing https://github.com/kubernetes-sigs/cluster-api/blob/main/.github/dependabot.yml#L10